Modix.Services/Csharp/DocumentationService.cs

```diff
@@ -2,12 +2,14 @@
 using System.Net.Http;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using System.Linq;
```

Comment:
I don't think we're actually using linq in this file, can probably remove the using directive.